xmtp/README.md

```diff
@@ -108,7 +127,53 @@ init():
     return
 ```
 
-### Receiving an invite
+### Receiving invites
+
+```plaintext
+downloadInvites():
+    Get the `refresh_jobs` record with an id of `invites`, and obtain a lock on the row:
+        Store `now()` in memory to mark the job execution start
+        Fetch all messages from invite topic with timestamp > refresh_job.last_run - PADDING_TIME # PADDING TIME accounts for eventual consistency of network. Maybe 30s.
```

Comment:
I thought about that. We do similar things in `xmtp-js`. The issue we run into is that with the eventual consistency of the network, we need to pad it by some number of seconds. That means we would be constantly reprocessing the most recent invite on subsequent calls (since the last timestamp - PADDING will always return at least one result).